tests/test_query_resource_path.py

```diff
@@ -132,13 +132,14 @@ def test_get_user_requests(client, access_token_user_only_patcher):
     assert res.status_code == 401, res.text
 
 
```

Comment:
it looks like these are all just updated so it checks the username without patching? I think you should add a test or two to explicitly test the new `azp` client_id code (unless I'm missing where that's at)

Reply:
The `access_token_user_client_patcher` fixture is used automatically, it tests both a user token and a client token.
The `access_token_user_only_patcher` fixture can be used explicitly in tests for endpoints that don't support client tokens.
Removing `access_token_user_only_patcher` means we fall back to testing both user tokens and client tokens

https://github.com/uc-cdis/requestor/blob/6cc3663/tests/conftest.py#L171-L215